app/dashboard/page.tsx

```
@@ -0,0 +1,383 @@
+                <Edit className="w-4 h-4" />
+              </Button>
+            </div>
+            <div className="flex items-center gap-4 text-sm text-amber-600">
+              <span>{formatDate(entry.date)}</span>
+              <span>{entry.wordCount} words</span>
+            </div>
+          </div>
+        </div>
+        <div
+          className="text-amber-800 text-sm leading-relaxed"
+          dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)